powerlevel9k.zsh-theme

```diff
@@ -1241,6 +1241,19 @@ prompt_dir_writable() {
   fi
 }
 
+# Kubernetes Current Context
+prompt_kubecontext() {
+  local kubectl=$(kubectl version 2>/dev/null)
+
+  if [[ -n "kubectl_version" ]]; then
```

Comment:
It was a copy and paste issue.. That's on me the `local kubectl` should be `local kubectl_version`. I will fix this!